src/db/migrations/20200418082540-add-organization-activity-address.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  QueryInterface,
+  DataTypes,
+} from "sequelize";
+
+export = {
+  up: (queryInterface: QueryInterface, Sequelize) => queryInterface.addColumn("organizations", "activity", {
+    type: new DataTypes.STRING(128),
+    allowNull: false,
+    defaultValue: "",
```

Comment:
why not allowing null instead?

Reply:
Because once the migration is done, we make it a required field in signup ! So we'll never have null values